doc/contributing.rst

```diff
@@ -285,6 +285,15 @@ or
 
 the first is preferred.
 
+When you specify an *RST role* (highlighting/interpretation marker) do it
+in the postfix form for uniformity, that is after \`text in backticks\`.
+For example an ``:idx:`` role for referencing a topic ("SQLite" in the
+example below) from `Nim Index`_ can be used in doc comment this way:
+
+.. code-block:: nim
+  ## A higher level `SQLite`:idx: database wrapper.
+
+.. _`Nim Index`: https://nim-lang.org/docs/theindex.html
```

Comment:
```suggestion
.. `Nim Index`_: https://nim-lang.org/docs/theindex.html
```

in the same "postfix" spirit?

Reply:
no, there is no such syntax in RST.

The idea is that you reference by writing `` link_ `` but set target by `` _link ``.

It would be impossible for inline targets to disambiguate these 2 cases.

Reply:
ok thanks!